R/html.r

```diff
@@ -0,0 +1,43 @@
+#' Returns HTML in a server.
+#'
+#' @param html_string character. A sring of HTML.
+#' @examples
+#' \dontrun{
+#'    routes <- list("/index" = function(...) html("<b>bold!</b>"))
```

Comment:
If you export the function, please remove the `\dontrun` block, since it should compile.

Also, you may wish to export an alternative name to avoid conflicting [with hadley](https://github.com/hadley/rvest/blob/master/NAMESPACE#L33) or at least mention when rvest and microserver are loaded at the same time.